.pre-commit-config.yaml

```diff
@@ -9,15 +9,11 @@ repos:
     - id: clippy
       args: ['--manifest-path=client/Cargo.toml', '--all', '--']
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.1.0"
+    rev: "v0.3.5"
     hooks:
       - id: ruff
         args: [ --fix]
-  - repo: https://github.com/psf/black
-    rev: 23.10.0
-    hooks:
-      - id: black-jupyter
-        exclude: ^(wandb/vendor/|wandb/proto/v.*/.*_pb2\.py|wandb/proto/v.*/.*_pb2_grpc\.py|wandb/proto/wandb_deprecated.py)
```

Comment:
will it format the auto generated files as well?

Reply:
or will it pick up the correct config from pyproject?

Reply:
the `pyproject.toml` config configures ruff to exclude generated files

Reply:
looks like it did the right thing! nice